v0/destinations/google_adwords_remarketing_lists/transform.js

```diff
@@ -0,0 +1,272 @@
+    const operations = {
+      create: {}
+    };
+    operations.create.userIdentifiers = element;
+    outputPayload.operations.push(operations);
+  });
+
+  return outputPayload;
+  /**
+   * This portion is to support remove and create both.
+   * If we need to support for remove too then need to uncomment this section
+   * and update addUserToJob function in util to handle add and remove both.
+   * Now we are directly sending the the body.JSON but to support remove we have to make two calls
+   * by setting data = body.JSON[0] and data = body.JSON[1] synultaneously.
+   */
+  // const outputPayloads = [];
+  // const typeOfOperation = Object.keys(listData);
+  // typeOfOperation.forEach(key => {
+  //   const userIdentidtifiersList = populateIdentifiers(
+  //     listData[`${key}`],
+  //     destination
+  //   );
+  //   const outputPayload = constructPayload(message, offlineDataJobsMapping);
+  //   outputPayload.operations = [];
+  //   // breaking the userIdentiFier array in chunks of 20
+  //   const userIdentifierChunks = returnArrayOfSubarrays(
+  //     userIdentidtifiersList,
+  //     20
+  //   );
+  //   // putting each chunk in different create/remove operations
+  //   userIdentifierChunks.forEach(element => {
+  //     const operations = {
+  //       [`${key}`]: {}
+  //     };
+  //     operations[`${key}`].userIdentifiers = element;
+  //     outputPayload.operations.push(operations);
+  //   });
+  //   outputPayloads.push(outputPayload);
+  // });
+  // return outputPayloads;
+};
+
+const processEvent = async (metadata, message, destination) => {
+  if (!message.type) {
+    throw new CustomError(
+      "[Google_adwords_remarketing_list]::Message Type is not present. Aborting message.",
+      400
+    );
+  }
+  if (!message.properties) {
+    throw new CustomError(
+      "[Google_adwords_remarketing_list]::Message properties is not present. Aborting message.",
+      400
+    );
+  }
+  if (!message.properties.listData) {
+    throw new CustomError(
+      "[Google_adwords_remarketing_list]::listData is not present inside properties. Aborting message.",
+      400
+    );
+  }
+  if (message.type.toLowerCase() === "audiencelist") {
+    const createdPayload = createPayload(message, destination);
+    return responseBuilder(metadata, createdPayload, destination);
+  }
+
+  throw new CustomError(
+    `[Google_adwords_remarketing_list]::Message Type ${message.type} not supported.`,
+    400
+  );
+};
+
+const process = async event => {
+  return processEvent(event.metadata, event.message, event.destination);
+};
+const processRouterDest = async inputs => {
+  if (!Array.isArray(inputs) || inputs.length <= 0) {
+    const respEvents = getErrorRespEvents(null, 400, "Invalid event array");
+    return [respEvents];
+  }
+
+  const respList = await Promise.all(
+    inputs.map(async input => {
+      try {
+        if (input.message.statusCode) {
+          // already transformed event
```

Comment:
this one is mainly required for backward compatibility. given that this is a new destination, do we actually need this?

Reply:
Removing this.